src/assets/js/installer.js

```diff
@@ -10,14 +10,28 @@
  * @return {void}
  */
 
-console.log( document.currentScript.attributes );
-( function( $, hooks, obj, namespace ) {
+( function( $, hooks, document ) {
 	'use strict';
+
+	/**
+	 * Initialize the necessary varaibles for StellarWP libraries such that
+	 * they are isolated from other instances of this library in the wild.
+	 */
+	// BEGIN: stellarwp library initialization.
+	const currentScript           = typeof document.currentScript !== 'undefined' ? document.currentScript : document.scripts[document.scripts.length - 1];
+	const namespace               = currentScript.getAttribute( 'data-stellarwp-namespace' );
```

Comment:
We should add a conditional to bail here with a `console.info()` about why it didn't initialize if the namespace check fails.